app/bundles/ReportBundle/Views/Report/form.html.php

```diff
@@ -136,7 +141,54 @@
                                 </div>
                             </div>
                         </div>
-
+                        <div class="tab-pane fade bdr-w-0" id="schedule-container">
+                            <div class="row">
+                                <div class="col-md-12">
+                                    <div class="pa-md">
+                                        <div class="checkbox">
+                                            <?php echo $view['form']->row($form['isScheduled']); ?>
+                                        </div>
+                                    </div>
+                                </div>
+                            </div>
+                            <div id="schedule_form">
+                                <div class="row">
+                                    <div class="col-md-6">
+                                        <div class="pa-md">
+                                            <?php echo $view['form']->row($form['toAddress']); ?>
+                                        </div>
+                                    </div>
+                                </div>
+                                <div class="row" id="schedule_form">
+                                    <div class="col-md-6">
+                                        <div class="pa-md">
+                                            <?php echo $view['form']->row($form['scheduleUnit']); ?>
+                                        </div>
+                                    </div>
+                                    <div class="col-md-6">
+                                        <div class="pa-md">
+                                            <div id='scheduleMonthFrequency''>
+                                                <?php echo $view['form']->row($form['scheduleMonthFrequency']); ?>
+                                            </div>
+                                            <div id='scheduleDay'>
+                                                <?php echo $view['form']->row($form['scheduleDay']); ?>
```

Comment:
This UI doesn't flow well. 
![image](https://user-images.githubusercontent.com/63312/32456332-dc3e3a78-c2ea-11e7-80bf-e6a407b8b2ab.png) "Every month last ON Tuesday" . Should be ok if we can get the ON above the last/first dropdown for this scenario.

Reply:
I can move it, but if you select week, there will be no "on" or twice "on" if I leave it as label too. That does not make sense.
Not sure what should I put to the labels so i works in both cases.